packages/angular/migrations.json

```diff
@@ -957,6 +957,10 @@
     "13.2.0": {
       "version": "13.2.0-beta.1",
       "packages": {
+        "jasmine-marbles": {
```

Comment:
This kind of update doesn't support dynamically determined versions which are reequired for jasmine-marbles since its version will be determined on the presence of RxJS 6 or 7. See the comment on the migration file.